src/kaleidoscope/plugin/LED-Wavepool.cpp

```diff
@@ -179,32 +178,29 @@ void WavepoolEffect::update(void) {
 #endif
 
   // draw the water on the keys
-  for (byte r = 0; r < ROWS; r++) {
-    for (byte c = 0; c < COLS; c++) {
-      uint8_t offset = (r * COLS) + c;
-      int8_t height = oldpg[pgm_read_byte(rc2pos + offset)];
+  for (auto key_addr : allKeyAddr()) {
+    int8_t height = oldpg[pgm_read_byte(rc2pos + key_addr.toInt())];
 #ifdef INTERPOLATE
-      if (now & 1) {  // odd frames only
-        // average height with other frame
-        height = ((int16_t)height + newpg[pgm_read_byte(rc2pos + offset)]) >> 1;
-      }
+    if (now & 1) {  // odd frames only
```

Comment:
Relatively minor, but the whitespace changes shouldn't be mixed in here.

Reply:
Ok.

Reply:
Just noticed that this is done by astyle because we got rid of one for loop, the indentation is automatically adapted. wontfix.